packages/cron/src/lib/fetch.js

```diff
@@ -5,9 +5,9 @@ import debug from 'debug'
 
 const log = debug('fetchJSON')
 
-const REQUEST_TIMEOUT = 60000
-const RETRY_INTERVAL = 60000
-const RETRY_ATTEMPTS = 5
+const REQUEST_TIMEOUT = 10000
+const RETRY_INTERVAL = 5000
+const RETRY_ATTEMPTS = 3
```

Comment:
@alanshaw dialling downt the retry logic on the pianta cron for an individual fail... we can always pick it up again on the next pass.